kvrocks.conf

```diff
@@ -863,5 +863,11 @@ rocksdb.write_options.memtable_insert_hint_per_batch no
 # Default: yes
 rocksdb.rate_limiter_auto_tuned yes
 
+# Support rocksdb `void_unnecessary_blocking_io` in iteration scenario
+# if user don't care iteration slowing by pruge blocking io, mark it no
+# see https://github.com/facebook/rocksdb/wiki/IO#avoid-blocking-io 
+# Default: yes
+# rocksdb.avoid_unnecessary_blocking_io yes
```

Comment:
```suggestion
# Enable this option will schedule the deletion of obsolete files in a background thread 
# on iterator destruction. It can reduce the latency if there are many files to be removed.
# see https://github.com/facebook/rocksdb/wiki/IO#avoid-blocking-io 
#
# Default: yes
# rocksdb.avoid_unnecessary_blocking_io yes
```